lib/ansible/modules/web_infrastructure/jira.py

```diff
@@ -243,7 +244,10 @@ def request(url, user, passwd, timeout, data=None, method=None):
     # resulting in unexpected results. To work around this we manually
     # inject the basic-auth header up-front to ensure that JIRA treats
     # the requests as authorized for this user.
-    auth = base64.encodestring('%s:%s' % (user, passwd)).replace('\n', '')
+    auth = base64.b64encode('{}:{}'.format(user, passwd).encode())
```

Comment:
This line is incompatible with py2.6, and should either continue using printf style string formatting, or use element positions such as `{0}:{1}` instead of `{}:{}`

Additionally, for proper types, please utilize `ansible.module_utils._text.to_text`.